integration-tests/lumberjack/pom.xml

```diff
@@ -51,6 +51,14 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-resteasy</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-bean</artifactId>
```

Comment:
Whenever you add camel extension dependencies into the itest pom, please run `mvn process-resources -Pformat -N` from the project root to regenerate the set of imported pom dependencies (E.g the stuff under the `The following dependencies guarantee that this module is built after them` comment).

Reply:
done !!
thanks !!